nomad/jitsi_packs/packs/jitsi_meet_web/templates/_custom-config.js.tpl

```diff
@@ -255,10 +255,13 @@ config.speakerStats = {
 if (!config.hasOwnProperty('whiteboard')) config.whiteboard = {};
 config.whiteboard.userLimit = [[ or (env "CONFIG_jitsi_meet_whiteboard_user_limit") "25" ]];
 
-[[ if eq (env "CONFIG_jitsi_meet_skip_interim_transcriptions") "true" -]]
 if (!config.hasOwnProperty('testing')) config.testing = {};
+[[ if eq (env "CONFIG_jitsi_meet_skip_interim_transcriptions") "true" -]]
 config.testing.skipInterimTranscriptions = true;
 [[- end ]]
+[[ if eq (env "CONFIG_jitsi_meet_mobile_xmpp_ws_threshold") "false" -]]
```

Comment:
```suggestion
[[ if ne (or (env "CONFIG_jitsi_meet_mobile_xmpp_ws_threshold") "false") "false" -]]
```
I believe you want to check if this has a value, and that this value isn't false.  I've defaulted the value to false above so that it will not be set unless the value is overridden to something other than false.